source/opt/function.h

```diff
@@ -103,6 +103,14 @@ class Function {
   void ForEachParam(const std::function<void(const Instruction*)>& f,
                     bool run_on_debug_line_insts = false) const;
 
+  // Pretty-prints all the basic blocks in this function into a std::string.
+  //
+  // Blocks are pretty-printed in reverse post-order.
```

Comment:
Recommendation from the paper authors.  Helps minimize the empty Phis the algorithm inserts.

Reply:
I'm sorry, I totally spaced out what part of the code you were referring to.  I thought it was the actual rewriter.  I did this in rpo because it's generally a good ordering for a CFG, but you are right about your comment on structured control flow, it may look wonky.  I'll change it.

Reply:
... And... looking at the code again, it doesn't even work in RPO!  It traverses in linear order using ForEachInst.  Sigh.  Comment updated.